assets/scss/_list.scss

```diff
@@ -0,0 +1,228 @@
+@use 'variables';
+
+.featured__wrapper {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  width: 100%;
+}
+
+.featured {
+  background: var(--mm-color-background);
+  border-radius: var(--mm-border-radius);
+  border-top: 1px solid rgba(0, 0, 0, 0.07);
+  box-shadow: 0 10px 15px -3px rgb(0 0 0 / 10%), 0 4px 6px -2px rgb(0 0 0 / 5%);
+  max-width: 800px;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+  z-index: 5;
+}
+
+.blog-post__container-featured {
+  display: grid;
+  gap: var(--mm-spacing);
+  grid-gap: var(--mm-spacing);
+  grid-template-areas:
+    'graphic'
+    'content';
+  padding: calc(var(--mm-spacing) / 2);
+}
+
+.blog-post__featured-image {
+  align-content: center;
+  border-radius: var(--mm-border-radius);
+  display: grid;
+  grid-area: graphic;
+  height: 100%;
+  justify-items: center;
+  overflow: hidden;
+  width: 100%;
+
+  @include variables.pattern;
+
+  svg,
+  img {
+    filter: drop-shadow(1px 2px 5px #0077c2);
+    margin: 0;
+    padding: calc(var(--mm-spacing) / 2);
+    position: relative;
+    width: 75%;
+    z-index: 10;
+  }
+}
+
+.blog-posts__wrapper {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  width: 100%;
+}
+
+.blog-posts {
+  display: grid;
+  gap: var(--mm-spacing);
+  justify-content: center;
+  max-width: var(--mm-max-width);
+  padding: calc(var(--mm-spacing) / 2);
+  width: 100%;
+}
+
+.blog-post__container {
+  display: grid;
+  margin: auto;
+  max-width: 650px;
+  padding: calc(var(--mm-spacing) / 2);
+}
+
+.content {
+  display: grid;
+  grid-area: content;
+  grid-template-areas:
+    'category'
+    'title'
+    'published'
+    'excerpt'
+    'readmore';
+  width: 100%;
+}
+
+.blog-post__categories {
+  display: flex;
+  flex-wrap: wrap;
+  font-family: var(--mm-font-stack-display);
+  font-size: 14px;
+  font-weight: 500;
+  grid-area: category;
+  margin-bottom: 10px;
+}
+
+.blog-post__category ~ .blog-post__category::before {
+  content: ' • ';
+}
+
+.blog-post__category:not(:last-child) {
+  padding-right: 4px;
+}
+
+.blog-post__heading {
```

Comment:
It'd be more appropriate if these `blog-post__**` classes lived in their own file.